app/views/content/how-to-apply-for-teacher-training/ske-explained-transcript.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Subject knowledge enhancement explained video transcript
+description: Full transcript for the 'Subject knowledge enhancement' video.
+---
+
+**Speaker 1:** SKE stands for subject knowledge enhancement and it allows you to build upon your subject knowledge that you would be expected to have as a teacher. And people do it if they maybe have an A level in the area but not a degree. 
+
+I personally didn’t have a degree in the area, so it felt quite intense and it was a lot of work, but I really loved doing it, and it’s allowed me to start my PGCE in physics with confidence that I know that I have the ability to teach that subject. 
+
+**Speaker 2: Michael Ashton, SKE Co-ordinator, Edge Hill University:** People perhaps have been in accountancy and think, well you know, “I would like to be a maths teacher” so there’s relevancy there. They might have a degree in maths but want to teach physics, so we would look at the whole person’s profile and then make a decision and recommend whether they needed subject knowledge enhancement. 
```

Comment:
There are two colons in the bold part (my fault!) - is there a better way of doing this?